test/refactoring/ConvertAsync/labeled_closure_params.swift

```diff
@@ -0,0 +1,59 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %refactor-check-compiles -add-async-alternative -dump-text -source-filename %s -pos=%(line+1):1 | %FileCheck -check-prefix=MULTIPLE-LABELED-RESULTS %s
+func mutlipleLabeledResults(completion: (_ first: String, _ second: String) -> Void) { }
+// MULTIPLE-LABELED-RESULTS: {
+// MULTIPLE-LABELED-RESULTS-NEXT: async {
+// MULTIPLE-LABELED-RESULTS-NEXT: let result = await mutlipleLabeledResults()
+// MULTIPLE-LABELED-RESULTS-NEXT: completion(result.first, result.second)
+// MULTIPLE-LABELED-RESULTS-NEXT: }
+// MULTIPLE-LABELED-RESULTS-NEXT: }
+// MULTIPLE-LABELED-RESULTS: func mutlipleLabeledResults() async -> (first: String, second: String) { }
+
+// RUN: %refactor-check-compiles -add-async-alternative -dump-text -source-filename %s -pos=%(line+1):1 | %FileCheck -check-prefix=MIXED-LABELED-RESULTS %s
+func mixedLabeledResult(completion: (_ first: String, String) -> Void) { }
+// MIXED-LABELED-RESULTS: {
+// MIXED-LABELED-RESULTS-NEXT: async {
+// MIXED-LABELED-RESULTS-NEXT: let result = await mixedLabeledResult()
+// MIXED-LABELED-RESULTS-NEXT: completion(result.first, result.1)
```

Comment:
I didn't know you could partially label a tuple!

Reply:
Neither did I. You can also still address `.first` as `.0`